browser/src/index.tsx

```diff
@@ -294,7 +294,7 @@ const start = (args: string[]) => {
     UI.init()
 
     ipcRenderer.on("menu-item-click", (_evt, message) => {
-        instance.command("normal! " + message)
+        instance.command("exec \":normal! " + message + "\"")
```

Comment:
Instead of using command() and then doing the escaping here, one could use `nvim_feedkeys()` (which takes same options as VimL `feedkeys()`). Assuming that's mapped to `instance.feedkeys()`,

    instance.feedkeys(message, "n", false)